Move pam.d and pam.conf man pages to section 5 where they belong.

diff -upk.orig Linux-PAM-0.80.orig/doc/Makefile Linux-PAM-0.80/doc/Makefile
--- Linux-PAM-0.80.orig/doc/Makefile	2004-09-22 09:37:47 +0000
+++ Linux-PAM-0.80/doc/Makefile	2005-08-03 15:13:50 +0000
@@ -100,6 +100,9 @@ remove:
 	cd man && for file in *.3 ; do \
 	  rm -f $(FAKEROOT)$(MANDIR)/man3/$$file ; \
 	done
+	cd man && for file in *.5 ; do \
+	  rm -f $(FAKEROOT)$(MANDIR)/man5/$$file ; \
+	done
 	cd man && for file in *.8 ; do \
 	  rm -f $(FAKEROOT)$(MANDIR)/man8/$$file ; \
 	done
@@ -139,10 +142,14 @@ ifeq ($(HAVE_SGML2HTML),yes)
 	done
 endif
 	mkdir -p $(FAKEROOT)$(MANDIR)/man3
+	mkdir -p $(FAKEROOT)$(MANDIR)/man5
 	mkdir -p $(FAKEROOT)$(MANDIR)/man8
 	for file in man/*.3 ; do \
 	  install -m 644 $$file $(FAKEROOT)$(MANDIR)/man3 ; \
 	done
+	for file in man/*.5 ; do \
+	  install -m 644 $$file $(FAKEROOT)$(MANDIR)/man5 ; \
+	done
 	for file in man/*.8 ; do \
 	  install -m 644 $$file $(FAKEROOT)$(MANDIR)/man8 ; \
 	done
diff -upk.orig Linux-PAM-0.80.orig/doc/man/pam.conf.5 Linux-PAM-0.80/doc/man/pam.conf.5
--- Linux-PAM-0.80.orig/doc/man/pam.conf.5	1970-01-01 00:00:00 +0000
+++ Linux-PAM-0.80/doc/man/pam.conf.5	2005-08-03 15:13:50 +0000
@@ -0,0 +1 @@
+.so man8/pam.8
diff -upk.orig Linux-PAM-0.80.orig/doc/man/pam.conf.8 Linux-PAM-0.80/doc/man/pam.conf.8
--- Linux-PAM-0.80.orig/doc/man/pam.conf.8	2000-12-28 16:31:50 +0000
+++ Linux-PAM-0.80/doc/man/pam.conf.8	1970-01-01 00:00:00 +0000
@@ -1 +0,0 @@
-.so pam.8
diff -upk.orig Linux-PAM-0.80.orig/doc/man/pam.d.5 Linux-PAM-0.80/doc/man/pam.d.5
--- Linux-PAM-0.80.orig/doc/man/pam.d.5	1970-01-01 00:00:00 +0000
+++ Linux-PAM-0.80/doc/man/pam.d.5	2005-08-03 15:13:50 +0000
@@ -0,0 +1 @@
+.so man8/pam.8
diff -upk.orig Linux-PAM-0.80.orig/doc/man/pam.d.8 Linux-PAM-0.80/doc/man/pam.d.8
--- Linux-PAM-0.80.orig/doc/man/pam.d.8	2000-12-28 16:31:50 +0000
+++ Linux-PAM-0.80/doc/man/pam.d.8	1970-01-01 00:00:00 +0000
@@ -1 +0,0 @@
-.so pam.8
